test/t/conftest.py

```diff
@@ -202,7 +202,7 @@ def bash(request) -> pexpect.spawn:
             INPUTRC="%s/config/inputrc" % testdir,
             TERM="dumb",
             LC_COLLATE="C",  # to match Python's default locale unaware sort
-            HISTFILE="/dev/null",  # to leave user's history file alone
+            HISTFILE="/tmp/bash_completion.bash_history",  # to leave user's history file alone
```

Comment:
If we end up writing to this file, I believe it will also be a file overwrite security issue through a symlink attack.

If we are to use a temporary file for this, we want to use a proper one created through either Python's `tempfile` facilities, or `mktemp` in shell (but that's not portable), and cleaned up appropriately. Unless we know of a real world situation in which we will end up writing there, I wouldn't personally bother but would just remove this and rely on the `unset`.

Reply:
> If we end up writing to this file, I believe it will also be a file overwrite security issue through a symlink attack.

Oh, you are right. Thank you.

Maybe I'm too nervous because I just lost my history of the recent few months, but I modified it to use `tempfile` 60c55437. Since `test/config/bashrc` is loaded after the first prompt is shown, the command history is already read when `unset -v HISTFILE` is executed. Maybe this doesn't affect the default user history file, but it is good to exclude dependencies of tests on the user's history file.

----

It seems that [the test for the new commit](https://github.com/scop/bash-completion/actions/runs/897929639) failed by timeout errors, but exactly the same commit has passed [the tests in Actions of my fork repository](https://github.com/akinomyoga/bash-completion/actions/runs/897929610).